.github/actions/build-rpm/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#Dockerfile to setup environment for centos rpm package 
+FROM node:14-alpine as builder
+WORKDIR /app
+COPY ./ /app
+RUN npm install && \
+    npm run build && \
+    npm run package
+
+FROM centos:latest
+USER 0
+# library need to generate rpm package
+RUN yum install -y rpm-build rpmdevtools gcc rpm-sign
+RUN curl -sL https://rpm.nodesource.com/setup_14.x | bash -
+RUN yum install -y nodejs
```

Comment:
Can we combine all RUN statements into one, since docker will make an extra layer for each RUN statement? 
[Reference](https://stackoverflow.com/a/39330458).